src/components/tools/Settings/SelectThresholdNumeric.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+  import { onMount, createEventDispatcher } from "svelte";
+  import { NumberInput, NumberInputSkeleton } from "carbon-components-svelte";
+  import { debounce } from "~/helpers/utilities";
+
+  export let value = 0;
+  export let title = "Select Threshold";
+
+  // TODO: these probably shouldn't be Infinity...
+  export let minValue = -Infinity;
```

Comment:
@mukhtyar any guidance on what the minimum and maximum temperature thresholds should be for HDD/CDD? I won't hard code them here but noticed if the threshold value is too low or too high the API won't return data.

Reply:
While 65 is standard and useful if you are comparing climates, I haven't come across any literature on what the range could be. More at [this link](https://www.degreedays.net/introduction#base-temperature-confusion). If we talk about heating & cooling buildings, I'd guess something like 40-100 are good thresholds (although extreme). Maybe instead of defining thresholds, we can show a fallback No Data message.